### PR TITLE
feat/add-menu-drop-down

### PR DESCRIPTION
**Pull request by ivangarcilazo**

### CODE REVIEW

**Comment by mohitkhatri06** [comment]
let other navBar options be there don't remove them,  expense and other options should be there

File: src/components/Header/Header.module.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    justify-content: center;
+    align-items: center;
+}
+.dropDown{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: white;
+
```
Comment (mohitkhatri06) [inline src/components/Header/Header.module.css:39]: box-shadow: 2px 2px 5px #aaaaaa;

**Comment by ivangarcilazo** [comment]
Oh sorry, I'll fix that now.

**Comment by ivangarcilazo** [comment]
Mohit, I check recently and the problem it's about de SIP page, I'll try fix that now anyways, but it's in master too. No from the header hehe, give me a moment and i can fix it.

**Review by mohitkhatri06 — APPROVED** [review]

**Comment by mohitkhatri06** [comment]
> Mohit, I check recently and the problem it's about de SIP page, I'll try fix that now anyways, but it's in master too. No from the header hehe, give me a moment and i can fix it.

thanks Ivan for resolving those.

**Pull request merged**